rollup.config.js

```diff
@@ -30,11 +30,13 @@ export default {
 		nodeResolve({browser: true}),
 		commonjs(),
 		postcss({ extensions: ['.css'], plugins: [postcss_url({url: 'inline'})] }),
-		copy({
-			targets: [
-				{ src: './manifest.json', dest: 'dist' },
-				{ src: './styles.css', dest: 'dist' }
-			]
-		})
+		...(process.env.BUILD !== 'development' ? [

```

Comment:
As above, this breaks how I work, as I'd like to copy the entirety of `dist` to Obsidian's plugin directory while testing as I develop. Do you suggest a different way to work?